src/main/web/src/scenes/Profile/ProfilePage.tsx

```diff
@@ -0,0 +1,80 @@
+    });
+
+    const [isEditing, setIsEditing] = useState<{ username: boolean, email: boolean }>({ username: false, email: false });
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const { name, value } = event.target;
+
+        // Sanitize input using DOMPurify
+        const sanitizedValue = DOMPurify.sanitize(value);
+
+        setProfileData({ ...profileData, [name]: sanitizedValue });
+    };
+
+    const handleEdit = (field: 'username' | 'email') => {
+        setIsEditing({ ...isEditing, [field]: !isEditing[field] });
+    };
+
+    const handlePasswordChange = () => {
+
+    };
+
+    return (
+        <div className="page">
+            <Header/>
+            <div className="profile-page">
```

Comment:
Div `profile-page` is not necessary, `page` already determines everything from this